README.md

```diff
@@ -377,6 +377,24 @@ file we have added a lot of comments into the code to make it easy to;
 
 Even with lots of comments we make to support more detailed documention.  You will find the documatation we have on the [Wiki pages] (https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/wiki).  Missing documentation you want?  Start a page and/or open an [issue] (https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/issues) to get it added.
 
+
+HOW TO VISUALIZE PerfKitBenchmarker RESULTS
+==================
+The dashboards directory contains a list of pre-built dashboards for every benchmark in PerfKitBenchmarker and a summary dashboard for all benchmarks. The dashboards can be used by PerfKitExplorer.
+
+To use pre-built dashboards
+* See PerfKitExplorer [README.md]
+(https://github.com/GoogleCloudPlatform/PerfKitExplorer/blob/master/README.md)
+for how to setup PerfKitExplorer and how to setup Google BigQuery dataset and table. 
+* Upload PerfKitBenchmarker results (/tmp/perfkitbenchmarker/run_*/perfkitbenchmarker_results.json) into BigQuery by running the following command (if you upload to a new BigQuery Table, a [schema]
+(https://github.com/GoogleCloudPlatform/PerfKitExplorer/blob/master/data/samples_mart/results_table_schema.json)
+file is needed):
+
+        bq load --source_format=NEWLINE_DELIMITED_JSON BIGQUERY_PROJECT_ID:BIGQUERY_DATASET.BIGQUERY_TABLE RESULTS_FILE (BIGQUERY_SCHEMA)
```

Comment:
The flags only works if the table already exists. Added the flag option and mentioned that it only works if the table exists.